### PR TITLE
PORT-11289 Bug | Change how MTTR is calculated

### PR DESCRIPTION
**Pull request by kodjomiles**
# Description

Changed `mttr` calculation to use `total average` and added admonition to filter records based on desired month for `lead time for change` and `mttr`

## Updated docs pages

Please also include the path for the updated docs

- Quickstart (`/`)
- Blueprint (`/platform-overview/port-components/blueprint`)
- Dora Metrics (`/guides/all/setup-dora-metrics#aggregation`)

### CODE REVIEW

**Review by Matanga1-2 — CHANGES_REQUESTED** [review]
In lead time, "measureTimeBy": "$createdAt", shouldn't it be "mergedAt"?
As you want to count based on when PRs were merged, not created.

**Comment by kodjomiles** [comment]
There is no mergedAt property when we use the ```total average```. we are not provided the option to select measureTimeBy. we are given that option when we select average of month,week,days or hours

**Review by Matanga1-2 — APPROVED** [review]

File: docs/guides/all/setup-dora-metrics.md [hunk 1]
```diff
@@ -1156,6 +1156,20 @@ Add this aggregation property to calculate the lead time for changes:
 Ensure that the target property is set to the correct Git provider (e.g., `githubPullRequest` for GitHub, `gitlabMergeRequest` for GitLab and `azureDevOpsPullRequest` for Azure DevOps).
```
Comment (hadar-co) [inline docs/guides/all/setup-dora-metrics.md:1156]: too many admonitions here.. The information under "Correct target property" should not be in an admonition

Comment (kodjomiles) [reply docs/guides/all/setup-dora-metrics.md:1156]: Okay

**Review by hadar-co — APPROVED** [review]

**Pull request merged**